bootstrap/utils/utils.go

```diff
@@ -161,3 +167,51 @@ func DeepCopy(src any, dest any) error {
 	}
 	return nil
 }
+
+// SendResponse puts together the response packet for the APIs
+func SendResponse(
```

Comment:
since it's hardcode to set the content type to JSON and json marshal, please rename to `SendJsonResp` in case we have other response type in the future